### ISSUE TITLE
 Floating invalid for ice_ocen_SIS2/SIS2_cgrid

### ISSUE THREAD

**Issue by adcroft** [NOAA-GFDL/SIS2#69]
Build [81](https://accessdev.nci.org.au/jenkins/job/mom-ocean.org/job/MOM6_run/81/) of Jenkins at NCI is reporting FPE/uninialized values for ice_ocen_SIS2/SIS2_cgrid.

From console:
```
forrtl: error (182): floating invalid - possible uninitialized real/complex variable.
MOM6               00000000020E4358  sis_dyn_cgrid_mp_        1356  SIS_dyn_cgrid.F90
MOM6               00000000025A4833  sis_dyn_trans_mp_         469  SIS_dyn_trans.F90
MOM6               0000000001187FFE  ice_model_mod_mp_         291  ice_model.F90
MOM6               0000000001184FEA  ice_model_mod_mp_         181  ice_model.F90
MOM6               000000000162A5C9  MAIN__                    879  coupler_main.F90
```

**Comment by adcroft** [comment]
@nicjhan I can't reproduce this with intel on Gaea. Any insight?

**Comment by Hallberg-NOAA** [comment]
This issue was corrected by commit NOAA-GFDL/SIS2@f0172c9b2fac2db12a8257bbcf470de940df84b.
